GitCommands/CommitMessageManager.cs

```diff
@@ -116,7 +135,16 @@ public string MergeOrCommitMessage
                         return;
                     }
 
-                    _fileSystem.File.WriteAllText(path, content, _commitEncoding);
+                    try
+                    {
+                        _fileSystem.File.WriteAllText(path, content, _commitEncoding);
+                    }
+                    catch (UnauthorizedAccessException)
+                    {
+                        MessageBox.Show(null, string.Format(_cannotAccessFile, path),
+                            _cannotSaveCommitMessage, MessageBoxButtons.OK,
+                            MessageBoxIcon.Exclamation, MessageBoxDefaultButton.Button1);
+                    }
```

Comment:
This look a duplicated code. I propose to move it out into an own method, so we provide a consistent experience across the module. 
...And, of course, we could test it (later as `MessageBox` isn't quite test-friendly) :)

Reply:
The message differs slightly

I could do with ignoring the message box completely...